application/libraries/BaseController.php

```diff
@@ -65,7 +65,7 @@ function isAdmin() {
 	 * This function is used to check the access
 	 */
 	function isTicketter() {
-		if ($this->role != ROLE_ADMIN || $this->role != ROLE_MANAGER) {
+		if ($this->role != ROLE_ADMIN && $this->role != ROLE_MANAGER) {
```

Comment:
This Code is wrong,
Test by login in admin and also manger account

Reply:
This is for showing only menus in the code. You can change the code as you want. This is just a demo.